Tutorials/whatiswot/5-JSON_Schema/5-JSON-Schema.vtt

```diff
@@ -0,0 +1,71 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:06.700 
+In this video, we will briefly explain Schema Languages and then move to the JSON Schema since it
+
+00:00:06.700 --> 00:00:13.400 
+is the one that describes and validates JSON, it is important to know in Web of Things scope.
+
+00:00:13.400 --> 00:00:16.675 
+Let us explain what is a schema.
+
+00:00:16.675 --> 00:00:24.531 
+Many payload formats offer a way to describe what a payload instance should look like.
+
+00:00:24.531 --> 00:00:32.387 
+Generally, these are called Schema Languages. For example, JSON Schema, XML Schema, PDF Schema.
+
+00:00:32.387 --> 00:00:35.875 
+Now let's take JSON Schema.
+
+00:00:35.875 --> 00:00:43.823 
+JSON Schema is a JSON document used to describe and validate how JSON payload instance should look like.
+
+00:00:43.823 --> 00:00:50.003 
+The main idea is to enable senders such as clients to understand how
+
+00:00:50.003 --> 00:00:56.182 
+the request should be and for the receiver to automatically validate the payloads.
```

Comment:
automatically pronunciation is weird, it puts too much emphasis on the beginning